src/generateUISchema.ts

```diff
@@ -156,6 +157,17 @@ const getUIElement = (key: string, schema: any, fieldSetItem: any = undefined) =
         options = getElementOptions(PropertyFormat.RepeatableField);
       }
       return getBaseUIObject(key, schema.schema.properties[key].title || '', options);
+
+    case undefined:
+    {
```

Comment:
That's to suppress linter warnings when there are declarations inside a case
`ESLint: Unexpected lexical declaration in case block. (no-case-declarations)`

We use it for `case SchemaTypes.String:` as well

Reply:
Ok, let's use same line